hedera-node/src/main/java/com/hedera/services/store/contracts/AbstractLedgerWorldUpdater.java

```diff
@@ -96,10 +98,12 @@
 
 	protected AbstractLedgerWorldUpdater(
 			final W world,
-			final WorldLedgers trackingLedgers
+			final WorldLedgers trackingLedgers,
+			final SideEffectsTracker sideEffectsTracker
```

Comment:
If I understand, this `sideEffectsTracker` isn't really an instance variable of the `Updater`---it should probably be an argument to the `wrappedTrackingLedgers()` method.